### ISSUE TITLE
Wallet: unit tests need for `fromSpendingKey*()` and `fromMasterKey()` static factories

### ISSUE THREAD

**Issue by msgilligan** [bitcoinj/bitcoinj#3042]
Neither the methods that take `Network` or the deprecated methods that take `NetworkParameters` are currently (directly, at least) tested.

**Comment by omoniyi24** [comment]
@msgilligan I’d like to take on this issue and implement unit tests for the Wallet factory methods (fromSpendingKeyB58() and fromMasterKey()), both of which use Network. I couldn’t find any methods using NetworkParameters, so my plan is to focus on these two methods only.

Please let me know if this aligns with your expectations or if there are additional methods or scenarios you’d like me to cover. Looking forward to your feedback!

**Comment by msgilligan** [comment]
You are correct, the methods that take `Network` are the right place to focus. The `NetworkParameters` methods were deprecated and have (I believe) been removed since this issue was opened.

Sounds like a great contribution. Good luck!

**Comment by omoniyi24** [comment]
@mswiggs I have created a PR that addresses this issue: [#3547](https://github.com/bitcoinj/bitcoinj/pull/3547).

Looking forward to your feedback.
